Sources/MotorControlUI/OverviewView.swift

```diff
@@ -0,0 +1,145 @@
+        }
+    }
+}
+
+struct OverviewNodeView: View {
+
+    @SwiftUI.Environment(\.surveyTintColor) var surveyTint: Color
+    @SwiftUI.Environment(\.spacing) var spacing: CGFloat
+    let bottomID = "bottom"
+    let overview: OverviewStepObject
+    
+    var body: some View {
+        GeometryReader { scrollViewGeometry in
+            ScrollViewReader { proxy in
+                ScrollView {
+                    VStack(alignment: .center, spacing: spacing) {
+                        if let imageInfo = overview.imageInfo {
+                            ContentImage(imageInfo)
+                                .background(surveyTint)
+                        }
+                        if let title = overview.title {
+                            Text(title)
+                                .font(.stepTitle)
+                                .foregroundColor(.textForeground)
+                        }
+                        if let subtitle = overview.subtitle {
+                            Text(subtitle)
+                                .font(.stepSubtitle)
+                                .foregroundColor(.textForeground)
+                                .multilineTextAlignment(.center)
+                        }
+                        if let detail = overview.detail {
+                            Text(detail)
+                                .font(.stepDetail)
+                                .foregroundColor(.textForeground)
+                                .multilineTextAlignment(.center)
+                        }
+                        if let icons = overview.icons {
+                            Text("This is what you'll need", bundle: SharedResources.bundle)
+                                .font(.stepIconHeader)
+                                .foregroundColor(.textForeground)
+                            HStack(alignment: .top, spacing: spacing) {
+                                ForEach(icons) { iconInfo in
+                                    VStack(alignment: .center, spacing: spacing) {
+                                        Image(iconInfo.icon, bundle: SharedResources.bundle)
+                                        Text(iconInfo.title)
+                                            .font(.stepIconText)
+                                            .foregroundColor(.textForeground)
+                                            .multilineTextAlignment(.center)
+                                            .fixedSize(horizontal: false, vertical: true)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    .id(bottomID)
+                    .onAppear{
+                        proxy.scrollTo(bottomID, anchor: .bottom)
+                    }
+                    .padding([.horizontal], spacing)
+                    .frame(maxWidth: scrollViewGeometry.size.width)
```

Comment:
Line 117 makes the screen start at the bottom of the ScrollView.
Line 119 is just padding so that the text isn't so close to the edges of the screen.
Line 120 prevents the overview image from expanding the entire view passed the screen's edges.

Reply:
Yeah, was wondering about that last line. Nice solution. Annoying that `.aspectRatio(contentMode: .fill).clipped()` doesn't work. *sigh*